client_dashboard/common/src/commonMain/kotlin/org/thechance/common/presentation/taxi/TaxiUiState.kt

```diff
@@ -66,3 +69,15 @@ fun Taxi.toUiState(): TaxiDetailsUiState = TaxiDetailsUiState(
 )
 
 fun List<Taxi>.toUiState() = map { it.toUiState() }
+
+data class AddTaxiDialogUiState(
+    val taxiPlateNumber: String = "",
```

Comment:
i've already said that taxiPlatenumber should be reffered to be plateNumber 
the data class already reffering to the vehicle type